### DISCUSSION TITLE
Love the new website

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1396] [General]
It looks really good and modern.

I especially love clarity of how everything is separated. The downloads section is especially cleared up a lot.

**Comment by vkbo** [comment]
Thanks!

It is built around a documentation theme that allows for adding multiple pages (or sections). I came across it when doing research for work, and I saw the potential. I also automated all of the integration with releases by adding a command line tool that will update the page by pulling data from the GitHub API. It makes it a lot easier to maintain as well as I don't have to do anything other than run a couple of commands for each release.
